src/types/index.ts

```diff
@@ -221,7 +222,7 @@ export interface INotification {
   createdAt: string
   entityId?: string
   entityType?: string
-  id?: number
+  id?: number | string
```

Comment:
I suggest to end all interface properties with semicolon like the TS docs are doing: https://www.typescriptlang.org/docs/handbook/2/objects.html#property-modifiers